lib/broccoli/ember-app.js

```diff
@@ -202,6 +199,13 @@ EmberApp.prototype._initOptions = function(options, isProduction) {
     }
   });
 
+  // Do not override custom css config if it was configured
+  this.options.outputPaths.app = Object.assign({
```

Comment:
If we are supporting Node 0.12.0 this will not work due to Object.assign not being a thing.